spec/imsc-hrm.html

```diff
@@ -410,26 +414,31 @@ <h3>Where is the HRM used?</h3>
       <h2>Architecture</h2>
 
       <figure id='fig-hypothetical-render-model'>
-        <img src="figures/hypothetical-render-figure-1.png" alt="Hypothetical Render Model">
+        <img src="figures/hypothetical-render-figure-1.svg" alt="Hypothetical Render Model">
 
         <figcaption>
           Hypothetical Render Model
         </figcaption>
       </figure>
 
       <p>The model illustrated in <a href="#fig-hypothetical-render-model"></a> operates on successive
-      <a data-lt="Intermediate Synchronic Document">Intermediate Synchronic Documents</a> obtained from an input <a>IMSC Document
-      Instance</a>, and uses a simple double buffering model: while non-empty <a>Intermediate Synchronic Document</a> E<sub>n</sub> is
-      being painted into Presentation Buffer P<sub>n</sub> (the "front buffer" of the model), the previous non-empty <a>Intermediate
-      Synchronic Document</a> E<sub>n-1</sub> is available for display in Presentation Buffer P<sub>n-1</sub> (the "back buffer" of
-      the model).</p>
-
-      <p>The model specifies a (hypothetical) time required for completely painting an <a>Intermediate Synchronic Document</a> as
-      a proxy for complexity. Painting includes drawing region backgrounds, rendering and copying glyphs, and decoding and copying
-      images. Complexity is then limited by requiring that painting of <a>Intermediate Synchronic Document</a> E<sub>n</sub>
-      starts at the presentation time of E<sub>n-1</sub> and completes by its presentation time. While <a>empty ISDs</a> are 
-      displayed, the time required to painting them is not considered significant, i.e., there is no inherent complexity in
-      painting and displaying the absence of ISD content.</p>
+      <a>Intermediate Synchronic Documents</a> E<sub>i</sub> obtained from an input <a>IMSC Document Instance</a>:</p>
+      <ul>
+        <li><a>non-empty ISDs</a> are processed using simple double buffering model: while a <a>non-empty ISD</a>
+        E<sub>n</sub> is being painted into a Front Buffer, the previous <a>non-empty ISD</a> E<sub>m</sub> is available
+        for display in a Back Buffer; and</li>
+        <li><a>empty ISDs</a> merely disconnect the Back Buffer from the display when presented,
+        i.e., presenting an <a>empty ISD</a> is equivalent to presenting nothing.</li>
+      </ul>
+
+      <p>The model specifies a (hypothetical) time required for completely painting a <a>non-empty ISD</a>
+      as a proxy for complexity. Painting includes clearing the Front Buffer, drawing region backgrounds,
+      rendering and copying glyphs, and decoding and copying images. Complexity is then limited by requiring that
+      painting of <a>non-empty ISD</a> E<sub>n</sub> starts at the presentation time of the previous non-empty
```

Comment:
timing language consistency nit, plus, painting can begin _after_ the presentation time of the previous non-empty ISD, if that ISD is presented for longer than `IPD`:

```suggestion
      painting of <a>non-empty ISD</a> E<sub>n</sub> begins no earlier than the presentation time of the previous non-empty
```